docs/cli.md

```diff
@@ -109,7 +148,49 @@ gcli metalake list
 #### Show a metalake details
 
 ```bash
-gcli metalake details --metalake metalake_demo 
+gcli metalake details --metalake metalake_demo
+```
+
+#### Create a metalake
+
+```bash
+gcli metalake create --metalake my_metalake --comment "This is my metalake"
+```
+
+#### Delete a metalake
+
+```bash
+gcli metalake delete --metalake my_metalake
+```
+
+#### Rename a metalake
+
+```bash
+gcli metalake update --metalake metalake_demo --rename demo
+```
+
+#### Update a metalake's comment
+
+```bash
+gcli metalake update --metalake metalake_demo --comment "new comment"
+```
+
+#### Display a metalake's properties
+
+```bash
+gcli metalake properties --metalake metalake_demo
+```
+
+#### Set a metalake's property
+
+```bash
+gcli metalake set --metalake metalake_demo --property test --value value
```

Comment:
I feel that "set-property" and "remove-property" are easier to understand, although they are longer. Say, except "property", Gravitino now supports "tag", which also have set-tag and remove-tag action.

Reply:
I didn't use set-property"set-property" or similar as they are too long and "--property" gives the context. Tags are handled in a different PR and also use set and remove but with "--tag". I don't think we need to change anything here. Think of it as saying this:  For this type of <entity> I will "set" a "--property" to this "--value".

Reply:
OK, I'm fine with it.